docs/tutorial/plotting_images.rst

```diff
@@ -0,0 +1,187 @@
+Plotting Images
+===============
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+This tutorial covers the built-in plotting functionality for images in Pylake. While most of the
+examples shown use kymographs, the details described here are valid for
+:meth:`Kymo.plot()<lumicks.pylake.kymo.Kymo.plot>`, :meth:`Scan.plot()<lumicks.pylake.scan.Scan.plot>`
+and :meth:`ImageStack.plot()<lumicks.pylake.ImageStack.plot>`.
+
+We'll start with loading some data as usual::
+
+    import lumicks.pylake as lk
+    filenames = lk.download_from_doi("10.5281/zenodo.7729525", "test_data")
```

Comment:
Import is not needed as it is already part of the assembled notebook when downloading.
```suggestion

    filenames = lk.download_from_doi("10.5281/zenodo.7729525", "test_data")
```